extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -28,6 +35,10 @@ public function getReviewedRevisionIdFromText( $pageName ) {
 		$currentRevisionModel = new Models\CurrentRevisionModel();
```

Comment:
I know it's not a part of this PR but I generally believe that we should use the controller to get this data (check the `getLatestReviewedRevision()` method).